WBooks/Library/MainMenuController.swift

```diff
@@ -58,7 +58,8 @@ extension MainMenuController: UITableViewDataSource, UITableViewDelegate {
     }
     
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        let controller = DetailViewController(viewModel: DetailViewModel(bookViewModel: _viewModel.books[indexPath.row]))
+        let bookViewModel = _viewModel.books[indexPath.row]
+        let controller = BookInfoViewController(viewModel: BookInfoViewModel(bookViewModel: bookViewModel), commentsController: BookCommentsViewController(viewModel: BookCommentsViewModel(bookViewModel: bookViewModel)), bookDetailsController: BookDetailsViewController(viewModel: BookDetailsViewModel(bookViewModel: bookViewModel)))
```

Comment:
This line is too long, break it in different lines or create the viewControllers in different properties and use them here.